languages/SMW_LanguageAr.php

```diff
@@ -52,62 +54,58 @@ class SMWLanguageAr extends SMWLanguage {
 
 	protected $m_SpecialProperties = array(
 		// always start upper-case
-		'_TYPE'  => 'لديه نوع',
-		'_URI'   => 'معرف الموارد الموحد معادلة',
-		'_SUBP' => 'الخاصية الفرعية ل',
-		'_SUBC' => 'تصنيف فرعي من',
-		'_UNIT' => 'عرض الوحدات',
-		'_IMPO' => 'المستوردة من',
-		'_CONV' => 'يقابل',
-		'_SERV' => 'يوفر الخدمة',
-		'_PVAL' => 'يسمح بالقيمة',
+		'_TYPE' => 'من النوع',
+		'_URI'  => 'URL مكافئ',
+		'_SUBP' => 'خصيصة فرعية من',
+		'_SUBC' => 'مُصنّف على التصنيف',
+		'_UNIT' => 'وحدة العرض',
+		'_IMPO' => 'مستوردة من',
+		'_CONV' => 'توافق',
+		'_SERV' => 'تقدّم خدمة',
+		'_PVAL' => 'تقبل القيمة',
 		'_MDAT' => 'تاريخ التعديل',
-		'_CDAT' => 'Creation date', // TODO: translate
-		'_NEWP' => 'Is a new page', // TODO: translate
-		'_LEDT' => 'Last editor is', // TODO: translate
-		'_ERRP' => 'يمتلك قيمة غير صحيحة ل',
-		'_LIST' => 'يمتلك حقول',
-		'_SOBJ' => 'Has subobject', // TODO: translate
-		'_ASK'  => 'Has query', // TODO: translate
-		'_ASKST'=> 'Query string', // TODO: translate
-		'_ASKFO'=> 'Query format', // TODO: translate
-		'_ASKSI'=> 'Query size', // TODO: translate
-		'_ASKDE'=> 'Query depth', // TODO: translate
-		'_ASKDU'=> 'Query duration', // TODO: translate
-		'_MEDIA'=> 'Media type',
-		'_MIME' => 'MIME type',
-		'_ERRC' => 'Has processing error',
-		'_ERRT' => 'Has processing error text',
-		'_PREC'  => 'Display precision of',
-		'_LCODE' => 'Language code',
-		'_TEXT'  => 'Text',
-		'_PDESC' => 'Has property description',
-		'_PVAP'  => 'Allows pattern',
-		'_DTITLE' => 'Display title of',
-		'_PVUC' => 'Has uniqueness constraint',
+		'_CDAT' => 'تاريخ الإنشاء',
+		'_NEWP' => 'هي صفحة جديدة',
+		'_LEDT' => 'آخر مَن حرّرها',
+		'_ERRP' => 'فيها قيمة غير صحيحة في',//الخصيصة
```

Comment:
>     I also know that the Arabic Wikipedia community has become
>     increasingly more conservative; not only in regard to localisation.
> 
> Maybe you are right. However, Arabic Wikipedia is the largest Arabic
> language wiki on the Internet. So, the opinion of the Arabic Wikipedia
> community should be taken into consideration here. After all, Semantic
> MediaWiki is a MediaWiki extension and it was we (Arabic Wikipedia
> users) who started the Arabic MediaWiki translation in the first place.

In my opinion it doesn't matter who *started* the translation in the
first place. Many first translations of software packages are worthless.
What I mean here is that first instance in itself should not be a
determinant.


>     I know they are used in Gnome, a major FOSS package, as well as the
>     Mozilla suite.
> 
> Both were translated by Arab Eyes people if I was not mistaken.

If this is so, does it automatically mean they are not orthodox?
I hope you're not governing on the work just by the identity of the
maker. If the point your putting forward is existing usability, then
existence in such popular software packages could mean these proposed
short-forms are probably used by many, regardless of how they got there.
Just like new words that pop-up everyday in all spoken languages. I
don't claim to have numbers, though.

> 
>     For me if is is useful, it should be allowed; people should be given
>     the chance to try them.
> 
> I respect your opinion but we disagree here. We simply don't "invent"
> language. In Wikipedia, we use only words and phrases commonly used in
> Arabic sources/literature.

I think you're carrying the "no original work" rule from Wikipedia
beyond its applicability here. All localisation of software and
technology entails original creation. Technology has already brought new
modes of language into usage. The Wikipedia jargon is itself an example
of that.


>     that it is an essential part of the Arabic writing system
> 
> Yes, I know. Both of us have more than 10 years experience with
> Wikipedia. However, you only make a few edits every month while I do
> make hundreds of edits every week. That's why we disagree here. The
> issue of diacritics is very annoying when are trying to find a certain
> word in 150 kilobytes article/webpage and you don't have time to read
> all the article/webpage. I agree that web browsers and search engines
> code should be modified as you described. However until this happens,
> let's not make things harder for very active users (who make most edits
> in any wiki as you know).

I think it's a matter of egg and chicken. If we put rules in place to
inhibit use because of limitations of technology, then it is less likely
for technology to solve.

Moreover, this difficulty could be valid in body text of articles. But
not in UI, right?

Regards,
Ahmad